packages/devtools-modules/client/shared/components/splitter/SplitBox.js

```diff
@@ -60,8 +60,9 @@ const SplitBox = React.createClass({
   getInitialState() {
     return {
       vert: this.props.vert,
-      width: this.props.initialWidth || this.props.initialSize,
-      height: this.props.initialHeight || this.props.initialSize
+      // We user integers for these properties
+      width: parseInt(this.props.initialWidth || this.props.initialSize),
+      height: parseInt(this.props.initialHeight || this.props.initialSize)
```

Comment:
Yes. But linting is limited to `devtools-launchpad` and `devtools-client-adapters` [https://github.com/devtools-html/devtools-core/blob/master/package.json#L9](https://github.com/devtools-html/devtools-core/blob/master/package.json#L9).